frontend/src/javascript/utils/axiosUtils.js

```diff
@@ -3,9 +3,8 @@ import toast, { getEndpointRelatedToast } from "@/javascript/utils/toastUtils";
 
 export function sendRequest(url, data = {}, method) {
     if (!validateMethod(method)) return;
-
+    axios.defaults.withCredentials = true;
```

Comment:
Rozumiem, że to wystarczy, że raz się wykona i wtedy wszystkie requesty mają dołączony ten header? Jeżeli tak to może wywołać to raz przy startowaniu całego projektu, np w main.js. Do sprawdzenia, czy działa poprawnie po tym przeniesieniu.